anime_downloader/sites/masterani.py

```diff
@@ -41,14 +41,12 @@ def _get_sources(self):
 
             ret.append((source['host']['name'].lower(), url))
 
-        # TODO: Clean this up. Why it looks like this, I don't know.
-        ret_1 = [(name, url) for name, url in ret if name in ['stream.moe']]
-        ret_1 += [(name, url) for name, url in ret if name == 'rapidvideo']
-        ret_1 += [(name, url) for name, url in ret if name == 'mp4upload']
+        sources = ['stream.moe', 'rapidvideo', 'mp4upload']
+        ret = [(name, url) for name, url in ret if name in sources]
```

Comment:
Did you test masterani after this change? If not, please do.

Reply:
Yes I did, works exactly the same way.